docs/development/Development.md

```diff
@@ -46,16 +46,15 @@ Note: Tests are written in C++ and linked with with firmware's C code.
 
 ### Running the tests.
 
-The tests and test build system is very simple and based of the googletest example files, it will be improved in due course.
+The tests and test build system is very simple and based of the googletest example files, it will be improved in due course. From the root folder of the project simply do:
```

Comment:
An opportunity to also correct the typo.
"Based off the googletest" not "Based of the googletest".